packages/security_detection_engine/kibana/security_rule/ad3f2807-2b3e-47d7-b282-f84acbbe14be.json

```
@@ -51,7 +51,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```

```suggestion
        "version": 12
```

Bumping as a result of bug identified: https://github.com/elastic/detection-rules/issues/1949